mlflow/ml-package-versions.yml

```diff
@@ -57,11 +57,10 @@ pytorch-lightning:
       pip install git+https://github.com/PytorchLightning/pytorch-lightning.git
 
   autologging:
-    minimum: "1.0.5"
+    minimum: "1.4.9"
```

Comment:
1.4.9 was released about two years ago:

https://pypi.org/project/pytorch-lightning/1.4.9/

Reply:
pytorch-lightning is not installed in MLR: https://docs.databricks.com/en/release-notes/runtime/13.3lts-ml.html